unity_pupil_plugin_vr/Assets/pupil_plugin/Scripts/Networking/Connection.cs

```diff
@@ -261,6 +261,19 @@ public void SetPupilTimestamp(float time)
 		}
 	}
 
+	public string GetPupilTimeStamp()
```

Comment:
```suggestion
	public string GetPupilTimestamp()
```
Consistency with `SetPupilTimestamp()`